pkgs/tools/audio/liquidsoap/full.nix

```diff
@@ -7,18 +7,23 @@
 
 let
   pname = "liquidsoap";
-  version = "2.2.5";
+  version = "2.3.x-unstable-2024-09-02";
 in
 stdenv.mkDerivation {
   inherit pname version;
 
   src = fetchFromGitHub {
     owner = "savonet";
     repo = "liquidsoap";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-o3P7oTizO2l2WkB4LormZ/Ses5jZOpgQ1r1zB1Y3Bjs=";
+    rev = "3e3ceaa958d2f678ecd32f370ae535b923c78d44";
+    hash = "sha256-0/8YfCU/hptIRNWR9+lYJ+2CVCrtijX+sB8utRCIRNQ=";
   };
 
+  patches = [
+    # <https://github.com/savonet/liquidsoap/pull/4105>
```

Comment:
Those comments are not markdown and I would rather prefer a comment about the patch.

Reply:
`<…>` is a much older plaintext convention for delineating links that Markdown adopted; that’s like complaining about `*emphasis*` in a comment. Anyway, I dropped these commits so it doesn’t matter.